website-custom-views/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -458,17 +471,17 @@ The history created by `renderApp` which is passed to the router. It can be used
 #### `user`
 
 **object**\
-The `user` object used to configure `<ApplicationContextProvider>`, so the result of merging the default user with `options.user`. Note that this is not the same as `applicationContext.user`. Can be `undefined` when no user is authenticated (when `options.user` was `null`).
+The `user` object used to configure `<CustomViewContextProvider>`, so the result of merging the default user with `options.user`. Note that this is not the same as `customViewContext.user`. Can be `undefined` when no user is authenticated (when `options.user` was `null`).
```

Comment:
@emmenko please double check if `ApplicationContextProvider` is used in Custom Views. I matched the values available in the CustomViewContext and they looked fine.